src/app/modules/side-menu/components/filter-menu/filter-menu.component.html

```diff
@@ -33,16 +33,32 @@
           <div slot="content">
             <ion-list lines="none">
               <!--if slush flow filter is set, disable filter by observation type because it cannot be combined -->
-              <ion-item *ngFor="let type of observationTypes$ | async; trackBy: obsTypeTrackById">
-                <ion-checkbox
-                  [checked]="!slushFlowFilterIsActive && type.isChecked"
-                  (click)="setNewType($event, type.parentId, type.id)"
-                  [disabled]="slushFlowFilterIsActive"
-                  labelPlacement="end"
-                  justify="start"
-                  >{{ type.name }}
-                </ion-checkbox>
-              </ion-item>
+              <!-- her vise obsgruppe først og så vise obstyper i gruppa og bruke klikks.-->
+              @for (group of this.groupsWithIsCheckedComputed(); track group?.id) {
```

Comment:
Du bør ikke trenge this. her og hvorfor trengs ? i track group.id, kan group være undefined? Ser jo ut som du filtrerer bort undefined i koden.